exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/transactions/ThreadLocalTransactionManager.kt

```diff
@@ -221,6 +248,20 @@ fun <T> inTopLevelTransaction(
                 if (repetitions >= repetitionAttempts) {
                     throw e
                 }
+                // set delay value
+                val delay = if (minRepetitionDelay < maxRepetitionDelay) {
+                    minRepetitionDelay + Random(System.currentTimeMillis()).nextLong(maxRepetitionDelay - minRepetitionDelay)
```

Comment:
Can this be simplified to `Random(System.currentTimeMillis()).nextLong(minRepetitionDelay, maxRepetitionDelay)`?